src/media/css/elements--select.styl

```diff
@@ -31,6 +31,10 @@ mkt-selected {
     display: flex;
     height: 60px;
     justify-content: center;
+
+    p {
+        padding: 0 10px; 
```

Comment:
Can you change this to `padding: 0 15px;` so it lines up with the text will line up with the top-left menu button.